source/uwp/SharedRenderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -289,74 +289,78 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering
             adaptiveCardElement, selectAction, renderContext, frameworkElement, XamlHelpers::SupportsInteractivity(hostConfig), true);
     }
 
-    template<typename TElement>
-    winrt::ImageSource XamlBuilder::SetImageOnUIElement(winrt::Uri const& imageUrl,
-                                                        winrt::AdaptiveCardResourceResolvers const& resolvers,
-                                                        ImageProperties<TElement> const& imgProperties)
+    winrt::Windows::Foundation::Size XamlBuilder::ParseSizeOfSVGImageFromString(winrt::hstring const& content)
     {
-        bool mustHideElement = true;
+        // Parse the size from the XamlDocument
```

Comment:
just so future us don't wonder if we were supposed to be using `XamlDocument` on the next line ;)

```suggestion
        // Parse the size from the XamlDocument as XML
```